docs/development/README.md

```diff
@@ -1 +1,13 @@
-This directory contains developer-oriented documentation.
+# Developer Documentation
+
+Note that the below table of contents may be out of date
+(we may forget to update it). If you don't find what you need,
+ask around!
+
+[Information About Factory Reset](docs/development/factory-reset.md)
```

Comment:
I believe all the links must be relative to the current directory, and not the repo root:

```suggestion
[Information About Factory Reset](../../docs/development/factory-reset.md)
```

It is possible that using absolute links work on Github, but would probably not work as well in a local IDE:

```suggestion
[Information About Factory Reset](/docs/development/factory-reset.md)
```

So I would prefer using relative links.